<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnyWaveClass</class>
 <widget class="QMainWindow" name="AnyWaveClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>475</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>AnyWave</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/AnyWave_logo.jpg</normaloff>:/images/AnyWave_logo.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainWinLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_files">
     <property name="title">
      <string>Recent files...</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuRecent_BIDS">
     <property name="title">
      <string>Recent BIDS directories</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_BIDS"/>
    <addaction name="actionOpen_video"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_New_AnyWave_Application"/>
    <addaction name="separator"/>
    <addaction name="actionImport_mrk_file"/>
    <addaction name="actionLoadBeamFormer"/>
    <addaction name="actionLoadICA"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave_display_to_PDF"/>
    <addaction name="actionExport_to_SVG"/>
    <addaction name="separator"/>
    <addaction name="actionFileProperties"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_files"/>
    <addaction name="menuRecent_BIDS"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMontage"/>
    <addaction name="actionMarkers"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView_">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionProcesses"/>
    <addaction name="actionPlugins"/>
    <addaction name="separator"/>
    <addaction name="actionDebug_Logs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuMapping">
    <property name="title">
     <string>Mapping</string>
    </property>
    <addaction name="actionShow_Mappings"/>
   </widget>
   <widget class="QMenu" name="menuICA">
    <property name="title">
     <string>ICA</string>
    </property>
    <addaction name="actionComponentsMaps"/>
    <addaction name="actionShow_map_on_signal"/>
   </widget>
   <widget class="QMenu" name="menuEpoch">
    <property name="title">
     <string>Epoch</string>
    </property>
    <addaction name="actionCreateEpoch"/>
    <addaction name="actionVisualiseEpoch"/>
    <addaction name="actionAveraging"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_AnyWave"/>
   </widget>
   <widget class="QMenu" name="menuProcesses">
    <property name="title">
     <string>Processes</string>
    </property>
    <addaction name="actionCreate_batch_script"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_new_MATLAB_plugin"/>
    <addaction name="actionCreate_new_Python_plugin"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAnyWave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView_"/>
   <addaction name="menuICA"/>
   <addaction name="menuMapping"/>
   <addaction name="menuEpoch"/>
   <addaction name="menuProcesses"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/fileopen-32.png</normaloff>:/images/fileopen-32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/ox_plugin_32.png</normaloff>:/images/ox_plugin_32.png</iconset>
   </property>
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
  <action name="actionMontage">
   <property name="text">
    <string>&amp;Montage</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/ox_application_exit_32.png</normaloff>:/images/ox_application_exit_32.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMarkers">
   <property name="text">
    <string>Markers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/ox_document_save_as_32.png</normaloff>:/images/ox_document_save_as_32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/ox_document_save_as_32.png</normaloff>:/images/ox_document_save_as_32.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCarto3D">
   <property name="text">
    <string>Mapping</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAbout_AnyWave">
   <property name="text">
    <string>About AnyWave</string>
   </property>
  </action>
  <action name="actionFileProperties">
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionMarkers_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Markers Bar</string>
   </property>
  </action>
  <action name="actionProcesses">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/system_run_32.png</normaloff>:/images/system_run_32.png</iconset>
   </property>
   <property name="text">
    <string>Processes</string>
   </property>
   <property name="toolTip">
    <string>Show Processes History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actExportChannelsInfo">
   <property name="text">
    <string>Export Channels Info.</string>
   </property>
  </action>
  <action name="actionExport_to_SVG">
   <property name="text">
    <string>Save display to SVG</string>
   </property>
  </action>
  <action name="actionImport_mrk_file">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/import_file_32.png</normaloff>:/images/import_file_32.png</iconset>
   </property>
   <property name="text">
    <string>Import marker file</string>
   </property>
  </action>
  <action name="actionStart_Matlab_Server">
   <property name="text">
    <string>Start Matlab Server</string>
   </property>
  </action>
  <action name="actionDebug_Logs">
   <property name="text">
    <string>Debug logs</string>
   </property>
  </action>
  <action name="actionShow_Mappings">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/carto_32x32.png</normaloff>:/images/carto_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Show mappings</string>
   </property>
  </action>
  <action name="actionImport_Layout_File">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/import_file_32.png</normaloff>:/images/import_file_32.png</iconset>
   </property>
   <property name="text">
    <string>Import layout file</string>
   </property>
  </action>
  <action name="actionExecuteScript">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/system_run_32.png</normaloff>:/images/system_run_32.png</iconset>
   </property>
   <property name="text">
    <string>Execute script</string>
   </property>
  </action>
  <action name="actionLoadICA">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/importICA32x32.png</normaloff>:/images/importICA32x32.png</iconset>
   </property>
   <property name="text">
    <string>Load ICA components</string>
   </property>
  </action>
  <action name="actionSave_display_to_PDF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save display to PDF</string>
   </property>
  </action>
  <action name="actionComponentsMaps">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/importICA32x32.png</normaloff>:/images/importICA32x32.png</iconset>
   </property>
   <property name="text">
    <string>Review components maps</string>
   </property>
  </action>
  <action name="actionShow_map_on_signal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show map on signal</string>
   </property>
  </action>
  <action name="actionLoad_Mesh">
   <property name="text">
    <string>Load Mesh</string>
   </property>
  </action>
  <action name="actionLoadBeamFormer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Beamformer matrix</string>
   </property>
  </action>
  <action name="actionCreateEpoch">
   <property name="text">
    <string>Edit conditions</string>
   </property>
  </action>
  <action name="actionVisualiseEpoch">
   <property name="text">
    <string>Review conditions</string>
   </property>
  </action>
  <action name="actionAveraging">
   <property name="text">
    <string>Averaging</string>
   </property>
  </action>
  <action name="actionOpen_BIDS">
   <property name="text">
    <string>Open BIDS</string>
   </property>
  </action>
  <action name="actiondrerz">
   <property name="text">
    <string>drerz</string>
   </property>
  </action>
  <action name="actionOpen_New_AnyWave_Application">
   <property name="icon">
    <iconset resource="../../resources/AnyWave.qrc">
     <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
   </property>
   <property name="text">
    <string>New AnyWave application</string>
   </property>
  </action>
  <action name="actionOpen_video">
   <property name="text">
    <string>Open video</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionICA_Batching">
   <property name="text">
    <string>ICA Batching</string>
   </property>
  </action>
  <action name="actionCreate_batch_script">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Batch Processing</string>
   </property>
  </action>
  <action name="actionHelpAnyWave">
   <property name="text">
    <string>AnyWave</string>
   </property>
  </action>
  <action name="actionHelpPlugins">
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
  <action name="actionCreate_new_MATLAB_plugin">
   <property name="text">
    <string>Create new MATLAB plugin</string>
   </property>
  </action>
  <action name="actionCreate_new_Python_plugin">
   <property name="text">
    <string>Create new Python plugin</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/AnyWave.qrc"/>
 </resources>
 <connections/>
</ui>
